server/graphql/v2/input/AccountReferenceInput.js

```diff
@@ -93,3 +94,72 @@ export const fetchAccountWithReference = async (
   }
   return collective;
 };
+/**
+ * Retrieves accounts for given ids or slugs
+ *
+ * @param {object} inputs - object containing slugs or ids of the collectives
+ * @param {object} params
+ *    - throwIfMissing: throws an exception if a collective is missing for a given id or slug
+ *    - whereConditions: additional where conditions to apply to the query
+ *    - attributes: to apply a SELECT on the query
+ *    - include: to include associated models
+ */
+export const fetchAccountsWithReferences = async (
```

Comment:
I took this one from @SudharakaP's work on https://github.com/opencollective/opencollective-api/pull/6595, but added a few teaks:
- Add a limit to prevent huge requests
- Add the ability to customize the query (include, attributes, etc)